src/applications/check-in/travel-claim/pages/complete/TravelClaimSuccessAlert.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { getUniqueFacilies } from '../../../utils/appointment';
+import { formatList } from '../../../utils/formatters';
+
+const TravelClaimSuccessAlert = props => {
+  const { appointments } = props;
+  const facilities = getUniqueFacilies(appointments);
```

Comment:
This is not going to be the final form of this check. We are going to store the facilities and times submitting for in context. So this will be a simple length check against that list. This is fine to get things moving, but I would add a @TODO here so we don't forget to change this.

Reply:
I didn't read your earlier comment about this before I posted this. But I added to that convo. Let me know if you want to chat this architecture out

Reply:
Added the todo